### PR TITLE
v2.3 changelog

### PR DESCRIPTION
**Pull request by wingo**

### CODE REVIEW

File: src/program/lwaftr/doc/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+   public IPv4 interface instead of being hairpinned if needed.
+
+ * Fix hairpinning of ICMP error messages for packets with the DF bit
+   that would cause fragmentation. Likewise these were always going out
+   the public interface.
+
+ * Allow B4s that have access to port 0 on their IPv4 address to be
+   pinged from the internet or from a hairpinned B4, and to reply.  This
+   enables a B4 with a whole IPv4 address to be pinged.  Having any
+   reserved ports on an IPv4 address will prevent pings.
```
Comment (kbara) [inline src/program/lwaftr/doc/CHANGELOG.md]: It's worth clarifying that those hosts can still initiate pings and get replies, just not be pinged themselves.

**Comment by kbara** [comment]
Other than clarifying the ping change, LGTM.

**Comment by wingo** [comment]
tx for the feedback, will fix

**Comment by kbara** [comment]
LGTM

**Pull request merged**